src/htmlrenderer.cpp

```diff
@@ -636,6 +740,7 @@ void HtmlRenderer::render(std::istream& input,
 			case HtmlTag::ITUNESHACK:
 			case HtmlTag::IMG:
 			case HtmlTag::HR:
+			case HtmlTag::SOURCE:
```

Comment:
The [specification ](https://html.spec.whatwg.org/multipage/embedded-content.html#the-source-element) says that `<source>` tags don't close.